js/common/modules/@arangodb/aql/explainer.js

```diff
@@ -1867,7 +1867,7 @@ function processQuery(query, explain, planIndex) {
       case 'RemoteNode':
         return keyword('REMOTE');
       case 'DistributeNode':
-        return keyword('DISTRIBUTE') + '  ' + annotation('/* create keys: ' + node.createKeys + ', variable: ') + variableName(node.variable) + ' ' + annotation('*/');
+        return keyword('DISTRIBUTE') + '  ' + annotation('/* variable: ') + variableName(node.variable) + ' ' + annotation('*/');
```

Comment:
Shall we change this to just
```suggestion
        return keyword('DISTRIBUTE') + ' ' + variableName(node.variable) ;
```

Reply:
Fine by me!

Reply:
Done